ansible/inventory/offline/group_vars/all/offline.yml

```diff
@@ -9,24 +9,34 @@ ansible_ssh_common_args: "{% if 'bastion' in groups['all'] %} -o ProxyCommand='s
 
 # This is used nowhere inside kubespray, only inside this file
 # and our own playbooks
-ubuntu_repos: "http://{{ assethost_host }}/debs/public"
+ubuntu_repos: "http://{{ assethost_host }}/debs-{{ ansible_distribution_release }}/public"
 ubuntu_repo_base_url: "{{ ubuntu_repos }}"
 ubuntu_repo_gpgkey: "{{ ubuntu_repos }}/gpg"
 
 docker_ubuntu_repo_base_url: "{{ ubuntu_repos }}"
 docker_ubuntu_repo_gpgkey: "{{ ubuntu_repos }}/gpg"
 # docker_ubuntu_repo_repokey: "9DC858229FC7DD38854AE2D88D81803C0EBFCD88"
 
-
 binaries_url: "http://{{ assethost_host }}/binaries"
+nodelocaldns_ip: 10.233.0.10
```

Comment:
Where does this IP come from?

Reply:
this was returned by kubesparay while creating a k8s cluster.... 
When I set this var, the ansible playbooks worked fine. Not sure if will change on each env